src/store/Workspaces/index.ts

```diff
@@ -317,6 +324,16 @@ export const reducer: Reducer<State> = (state: State | undefined, action: KnownA
         isLoading: false,
         workspaces: state.workspaces.map(workspace => workspace.id === action.workspace.id ? action.workspace : workspace),
       });
+    case 'UPDATE_WORKSPACE_STATUS':
+      return createState(state, {
+        workspaces: state.workspaces.map(workspace => {
+          if (workspace.id !== action.workspaceId) {
+            return workspace;
+          }
+          workspace.status = action.status;
+          return workspace;
+        })
```

Comment:
I think it can be simpler:
```
       workspace => {
          if (workspace.id !== action.workspaceId) {
            return workspace;
          }
          workspace.status = action.status;
          return workspace;
        })
```
=>

```
       workspace => {
          if (workspace.id === action.workspaceId) {
            workspace.status = action.status;
          }
          return workspace;
        })
```

Reply:
good point, thanks!